rasa/core/policies/policy.py

```diff
@@ -226,6 +302,9 @@ def predict_action_probabilities(
             domain: the :class:`rasa.shared.core.domain.Domain`
             interpreter: Interpreter which may be used by the policies to create
                 additional features.
+        Keyword Args:
```

Comment:
let's add these as proper parameters in the method signature
```suggestion
```

Reply:
then pycharm complains